src/locale/languages.ts

```diff
@@ -58,7 +59,8 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.it, name: 'Italiano – Italian'},
   {code2: AppLanguage.ja, name: '日本語 – Japanese'},
   {code2: AppLanguage.ko, name: '한국어 – Korean'},
-  {code2: AppLanguage.nl, name: 'Nederlands – Dutch'},
+  {code2: AppLanguage.nl, name: 'Nederlands – Dutch' },
```

Comment:
```suggestion
  {code2: AppLanguage.nl, name: 'Nederlands – Dutch'},
```
An extra space seems to have been added here by accident